recipe/activate-clang.sh

```diff
@@ -154,6 +154,7 @@ fi
 
 _tc_activation \
   activate @CHOST@- "HOST,@CHOST@" \
+  "CONDA_TOOLCHAIN_HOST,@CHOST@" \
```

Comment:
```suggestion
  "CONDA_TOOLCHAIN_HOST,@CHOST@" \
  "CONDA_TOOLCHAIN_BUILD,@CBUILD@" \
```

Reply:
This needs to be in the deactivate script too.